settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
+}
```

Comment:
However, the `formatOnSave` doesn't seem to work. I think this is because this setting should live in `.vscode/settings.json` (your "workspace settings") instead. I added it there and it formats on save as expected. 👍  

However, that file is gitignored. It seems like folks have mixed opinions on whether it's OK to commit this file, as at some level it's a user's settings for their editor. https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control

I'm fine with committing it (would first need to remove from `.gitignore`) or otherwise just adding a note suggesting `formatOnSave: true` in the readme.

Reply:
Ooooh gotcha. I'll make a note of this in the Readme and delete this file. No wonder it didn't work for me the first time 😛